lib/connection/connection_config.js

```diff
@@ -483,6 +483,14 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
     disableQueryContextCache = options.disableQueryContextCache;
   }
 
+  let maxLoginTimeout = 300;
+  if (Util.exists(options.maxLoginTimeout)) {
+    Errors.checkArgumentValid(Util.isNumber(options.maxLoginTimeout),
```

Comment:
The login timeout can only be increased so it should also check if the value is lower than the default value

Reply:
Add checking.